tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -174,7 +173,11 @@ def prepare_ir(argv: argparse.Namespace):
         moc_front_end.add_extension(TelemetryExtension("ovc", t.send_event, t.send_error, t.send_stack_trace))
         if any_extensions_used(argv):
             for extension in argv.extension:
-                if not isinstance(extension, ModuleExtension):
+                try:
+                    from openvino.frontend.pytorch.module_extension import ModuleExtension
+                    if not isinstance(extension, ModuleExtension):
+                        moc_front_end.add_extension(extension)
```

Comment:
If PT FE is not available, then the import fails and no extensions will be registered for other FEs.